ddl/index_merge_tmp_test.go

```diff
@@ -518,13 +519,7 @@ func TestAddIndexMergeConflictWithPessimistic(t *testing.T) {
 		tk.MustExec("alter table t add index idx(a);")
 		afterCommit <- struct{}{}
 	}()
-	timer := time.NewTimer(300 * time.Millisecond)
-	select {
-	case <-timer.C:
-		break
-	case <-afterCommit:
-		require.Fail(t, "should be blocked by the pessimistic txn")
-	}
+	<-doRollbackCh
```

Comment:
Why is it changed? The original logic is to make sure adding index cannot complete in 300ms because the pessimistic DML is active.